### DISCUSSION TITLE
Science and technology 🧪🔬

### DISCUSSION THREAD

**Discussion by Sajansharma0017** [ParanoidUser/yolo#1206] [Galaxy Trivia] [answered]
Who is known as the "Missile Man" of INDIA?

**Comment by tomerf** [comment, marked as answer]
A. P. J. Abdul Kalam came to be known as the Missile Man of India for his work on the development of ballistic missile and launch vehicle technology.

**Comment by Sajansharma0017** [comment]
Congratulations you're right!
